pkg/resource/persistence/schema_scripts_configmap_builder.go

```diff
@@ -262,6 +250,60 @@ func (b *SchemaScriptsConfigmapBuilder) Update(object client.Object) error {
 		baseData.MTLSProvider = string(b.instance.Spec.MTLS.Provider)
 	}
 
+	defaultStoreTool := "temporal-sql-tool"
+	var renderedCreateDefaultDatabase bytes.Buffer
+	var renderedCreateVisibilityDatabase bytes.Buffer
+	if defaultStoreType == v1alpha1.CassandraDatastore {
+		// Fix for https://github.com/temporalio/temporal/blob/master/tools/cassandra/main.go#L70
+		// Which requires an env var set.
+		defaultStoreTool = "CASSANDRA_PORT=9042 temporal-cassandra-tool"
+
+		err = templates[CreateCassandraTemplate].Execute(&renderedCreateDefaultDatabase, createKeyspace{
+			baseData:       baseData,
+			Tool:           defaultStoreTool,
+			ConnectionArgs: b.argsMapToString(defaultStoreArgs),
+			KeyspaceName:   defaultStore.Cassandra.Keyspace,
+		})
+		if err != nil {
+			return fmt.Errorf("can't render create-default-keyspace.sh: %w", err)
+		}
+	} else {
+		err = templates[CreateDatabaseTemplate].Execute(&renderedCreateDefaultDatabase, createDatabase{
+			baseData:       baseData,
+			Tool:           defaultStoreTool,
+			ConnectionArgs: b.argsMapToString(defaultStoreArgs),
+			DatabaseName:   defaultStore.SQL.DatabaseName,
+		})
+		if err != nil {
+			return fmt.Errorf("can't render create-default-database.sh: %w", err)
+		}
+	}
+
+	visibilityStoreTool := "temporal-sql-tool"
+	if visibilityStoreType == v1alpha1.CassandraDatastore {
+		visibilityStoreTool = "CASSANDRA_PORT=9042 temporal-cassandra-tool"
+
+		err = templates[CreateCassandraTemplate].Execute(&renderedCreateVisibilityDatabase, createKeyspace{
+			baseData:       baseData,
+			Tool:           defaultStoreTool,
+			ConnectionArgs: b.argsMapToString(defaultStoreArgs),
+			KeyspaceName:   defaultStore.Cassandra.Keyspace,
+		})
+		if err != nil {
+			return fmt.Errorf("can't render create-visibility-keyspace.sh: %w", err)
+		}
+	} else {
+		err = templates[CreateDatabaseTemplate].Execute(&renderedCreateVisibilityDatabase, createDatabase{
+			baseData:       baseData,
+			Tool:           defaultStoreTool,
```

Comment:
```suggestion
			Tool:           visibilityStoreTool,
```

Reply:
now i look like a chump so much for first impressions

Reply:
No problem, you're new to the codebase, which is perhaps a bit complicated. 
As I'm currenlty the only maintainer, the current codebase lacks of code reviews which inevitably makes the code harder to understand for others :)